LEGO1/mxeventpresenter.h

```diff
@@ -24,13 +24,17 @@ class MxEventPresenter : public MxMediaPresenter {
 		return !strcmp(name, MxEventPresenter::ClassName()) || MxMediaPresenter::IsA(name);
 	}
 
-	virtual MxResult AddToManager() override; // vtable+0x34
-	virtual void Destroy() override;          // vtable+0x38
+	virtual void ReadyTickle() override;           // vtable+0x18
+	virtual void StartingTickle() override;        // vtable+0x1c
+	virtual MxResult AddToManager() override;      // vtable+0x34
+	virtual void Destroy() override;               // vtable+0x38
+	virtual undefined4 PutData() override;         // vtable+0x4c
+	virtual void CopyData(MxStreamChunk* p_chunk); // vtable+0x5c
 
 private:
 	void Init();
 
-	undefined4* m_unk50;
+	MxU8* m_data; // 0x50
```

Comment:
Question: Were (and I guess, are, since I'm focusing on Java for AP CS), array declarations such as: `MxU8 m_data[];` allowed? This way its specifically being an array

Reply:
That is not valid C++. You can declare an array with a static size but that's about it. Java has different fundamentals and while some of the syntax may be similar, it's not applicable here.